README.md

```diff
@@ -150,8 +150,14 @@ The RSC code is split into two separate codebases: the library itself which live
 ## Building
 
 ### Required Software
-Node 10.x or 12.x.  Node 14.x is known not to work currently.
-yarn 1.21.1
+<!-- Node 10.x or 12.x. Node 14.x is known not to work currently. -->
+Node 12.x
```

Comment:
This is out of date even before this PR.  We're on Node 16 now and also support 14